analyzer/metadata-plugin/src/main/resources/metadata_db_c_init.sql

```diff
@@ -0,0 +1,182 @@
+CREATE TABLE packages
```

Comment:
I don't think we need this file as it is an exact duplicate of the `metadata_db_init.sql` which already in the same directory.

Reply:
See lines 138, 167.

Reply:
Got it, my bad. Then the file is indeed needed.